### ISSUE TITLE
Canada.ca theme: Background color is being overwritten 

### ISSUE THREAD

**Issue by parsbra** [wet-boew/GCWeb#1011]
See: #6987 - https://github.com/wet-boew/wet-boew/issues/6987

What is the intended background color for the body of the page?
- <b>theme.min.css</b> ­­> has: 

@media screen
body {
background: #f9f9f9;}

body {
background: #f8f8f8;}
- <b>wet-boew.min.css</b> > has:

body {
background-color: #fff;}

Currently the background color being used is #f9f9f9 because theme.min.css appears last in the HTML, but I am guessing that it should be white for the body.

**Comment by parsbra** [comment]
@LaurentGoderre whats the turn around time on a response for these issues? it's been 8 days...

**Comment by LaurentGoderre** [comment]
@masterbee please respond.

**Comment by masterbee** [comment]
@parsbra - thanks for question and the background color has been #f8f8f8, since it's conception. Other themes like the Base Theme and WET theme have different background colors hence the flexibility of the theme-ing .

**Comment by parsbra** [comment]
@masterbee -  well it's using f9f9f9. My question is,  the Canada theme has both CSS files theme and wet, shouldn't the wet CSS,  white body,  be the correct one?

**Comment by LaurentGoderre** [comment]
Not necessarily, the theme is free to override whatever styles that WET defines.

**Comment by parsbra** [comment]
The theme CSS is only overwriting it because it appears after the wet CSS file.  If you switch it so the wet file is last the background  is white.  
This seems more like a CSS conflict rather than a conscience decision to change it from white to light grey. 
 The background has always been white in the past which is why I'm trying to figure this out.  All of our images, mainly charts,  have white backgrounds.

**Comment by LaurentGoderre** [comment]
By design, the theme is after so it can override styles. In the latest version, this is not an issue because the two stylesheets are merged together. 

I did not design this theme so i can't say for sure what the intent is but it appears to me that is was designed that way

**Comment by masterbee** [comment]
@parsbra - The CSS is stacked correctly and the theme was originally created with the f8f8f8 color. I can understand that it looks incorrect but that was the design color for the theme.

**Comment by parsbra** [comment]
@masterbee it's using f9f9f9, because f8f8f8 is being overwritten as per below: (both are in theme)

@media screen
body {
background: #f9f9f9;}

body {
background: #f8f8f8;}

**Comment by shawnthompson** [comment]
@masterbee he's right the body is #f9f9f9 according to the [_screen.scss](https://github.com/wet-boew/GCWeb/blob/a9830b4ad8bd3b8a3a2ddf07b33215613094325c/src/views/_screen.scss#L6) file.

Does this answer your question @parsbra. That the Canada theme's background is #f8f8f8 by default, although it is set to #f9f9f9 which we will be changing soon.

**Comment by hsrudnicki** [comment]
@parsbra and others:  Is this above still an issue? Either way, please update this issue's status, whether this issue can be closed as resolve/fixed based on last comment(s) or otherwise?  Thank you

**Comment by hsrudnicki** [comment]
Is this still an issue? Can be this issue closed or otherwise? Please update the status of this for the current year, thank you
